src/docs/components/DocsActiveBlock.jsx

```diff
@@ -48,25 +50,39 @@ class DocsActiveBlock extends Component {
     });
   }
 
+  remountComponent() {
+    this.setState({renderNormally: false}, () => this.setState({renderNormally: true}));
+  }
+
   render() {
-    const component = React.cloneElement(this.props.children, this.state.props);
+    let component;
     let code;
 
-    if (this.state.showCode === 'jsx') {
-      const jsx = generateJSX(component);
+    if (this.state.renderNormally) {
+      component = React.cloneElement(this.props.children, this.state.props);
+
+      if (this.state.showCode === 'jsx') {
+        const jsx = generateJSX(component);
 
-      code = <DocsBlock><CodeBlock type="jsx">{jsx}</CodeBlock></DocsBlock>;
-    } else if (this.state.showCode === 'html') {
-      const html = ReactDOMServer.renderToStaticMarkup(component);
+        code = <DocsBlock><CodeBlock type="jsx">{jsx}</CodeBlock></DocsBlock>;
+      } else if (this.state.showCode === 'html') {
+        const html = ReactDOMServer.renderToStaticMarkup(component);
 
-      code = <DocsBlock><CodeBlock type="html">{html}</CodeBlock></DocsBlock>;
+        code = <DocsBlock><CodeBlock type="html">{html}</CodeBlock></DocsBlock>;
+      }
     }
 
     const blockClass = classnames('docs-active-block', {
       'docs-active-block--gray': this.state.changeBackground === 'light',
       'docs-active-block--dark': this.state.changeBackground === 'dark'
     });
 
+    component = [this.props.contentBefore, component, this.props.contentAfter];
+
+    if (this.props.wrapper) {
+      component = React.cloneElement(this.props.wrapper, {}, component);
+    }
```

Comment:
`contentBefore` is injected before the component, `contentAfter` after component and `wrapper` wrapps `[contentBefore, component, contentAfter]`